### PR TITLE
Members

### PR DESCRIPTION
**Pull request by tomkr**
This PR adds the members function, which is useful for getting Clips from Playlists for instance.

### CODE REVIEW

File: lib/poms.rb [hunk 1]
```diff
@@ -31,6 +31,12 @@ def descendants(mid, search_params)
     JSON.parse(request.execute.body)
   end
 
+  def members(mid)
+    assert_credentials
```
Comment (mxgrn) [inline lib/poms.rb]: Can't we move this check out to `GetRequest`?

Comment (rpbaptist) [inline lib/poms.rb]: Absolutely. We are doing that now.

Comment (mxgrn) [inline lib/poms.rb]: Do you mean it'll be a separate PR? Perfect.

**Comment by mxgrn** [comment]
Looks good as far as I can understand what it does.
